include/coap/coap_session.h

```diff
@@ -317,5 +320,115 @@ coap_session_t *coap_session_get_by_peer(struct coap_context_t *ctx,
 
 void coap_session_free(coap_session_t *session);
 
+ /**
+  * @defgroup cc Rate Control
```

Comment:
I had moved them so to expose them, so that the application could reset them to the default values if needed.

Reply:
Good point. Should we drop the `#ifndef` guards?
This could break existing applications (I introduced this some time ago for someone who wanted be able to change the defaults) but with the sessions we already have a huge API change...

Reply:
I think that the #ifndef guards should remain for backwards compatibility, otherwise there will be compilation issues if someone has redefined them.

Reply:
Fine by me (this solution has issues by itself but you may be correct that backwards-compatibility is more important).

Reply:
Actually, the more I think about the #ifndef, they ought to be removed,  otherwise there is the potential for the library to be compiled with one version and the application with another creating potential confusion.  As the updated API provides the necessary flexibility, then moving forward, that should be used for any application compilations that fail.

Reply:
That is the reason why I asked this in the first place. But we have the same situation in `pdu.h` for years. Therefore, I can live with both and leave it up to you to drop it or not.